Libs/Widgets/ctkDirectoryButton.cpp

```diff
@@ -74,7 +74,8 @@ ctkDirectoryButtonPrivate::ctkDirectoryButtonPrivate(ctkDirectoryButton& object)
 void ctkDirectoryButtonPrivate::init()
 {
   Q_Q(ctkDirectoryButton);
-  this->PushButton = new QPushButton(q);
+  this->PushButton = new ctkPushButton(q);
+  this->PushButton->setElideMode(Qt::ElideMiddle);  // truncate the middle of the path if does not fit
```

Comment:
documenttion says that it is none by default

Reply:
fixed